### ISSUE TITLE
Master is currently broken due to bad import

### ISSUE THREAD

**Issue by chr15m** [piku/piku#154]
Looks like this snuck in during b5a4e79e - piku.py from master is currently throwing this:

```
Traceback (most recent call last):
  File "/home/piku/piku.py", line 16, in <module>
    from os import chmod, getgid, getuid, getgrgid, symlink, unlink, remove, stat, listdir, environ, makedirs, O_NONBLOCK
ImportError: cannot import name 'getgrgid'
Connection to cloud.mccormickit.com closed.
```

**Comment by rcarmo** [comment]
Hmmm. I added those changes to fix #150, and it is working on the machine I fixed it in. Let me see what it is running...

**Comment by rcarmo** [comment]
So I patched the wrong lines when updating on my iPad. Such is mobile life. 

For good measure, testing/linting now runs `piku` once before linting.
